### PR TITLE
PUD-1071 - Change history page - confirm who uploaded the doc

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: src/test/java/cucumber/steps/NavigationSteps.java [hunk 1]
```diff
@@ -664,6 +664,18 @@ public void viewDossierCreationDetails(String caseworker) {
         );
     }
 
+    @When("{word} navigates to view the change history for the recall")
+    public void navigateToChangeHistory(String caseworker) {
+        Actor actor = theActorCalled(caseworker);
+        String recallId = actor.recall("RECALL_ID");
```
Comment (philsegal) [inline src/test/java/cucumber/steps/NavigationSteps.java:670]: do we need this?

File: src/test/java/cucumber/steps/NavigationSteps.java [hunk 2]
```diff
@@ -664,6 +664,18 @@ public void viewDossierCreationDetails(String caseworker) {
         );
     }
 
+    @When("{word} navigates to view the change history for the recall")
+    public void navigateToChangeHistory(String caseworker) {
+        Actor actor = theActorCalled(caseworker);
```
Comment (philsegal) [inline src/test/java/cucumber/steps/NavigationSteps.java:669]: assuming we dont need the line below, should inline `actor` too

File: src/test/java/cucumber/steps/NavigationSteps.java (hunk 1 above)
Comment (jonnywyatt) [reply src/test/java/cucumber/steps/NavigationSteps.java]: no, good spot

File: src/test/java/cucumber/steps/NavigationSteps.java (hunk 2 above)
Comment (jonnywyatt) [reply src/test/java/cucumber/steps/NavigationSteps.java]: done

**Review by philsegal — APPROVED** [review]

**Pull request merged**